### PR TITLE
Fix Test Setup

### PR DESCRIPTION
**Pull request by halbekanne**

### CODE REVIEW

File: src/app/features/home/components/login-form/login-form.component.spec.ts [hunk 1]
```diff
@@ -40,13 +56,13 @@ describe('LoginFormComponent', () => {
...
   });
 
-  describe('the user enters existing user credentials and submits the form', () => {
-
-    beforeEach(() => {
-
-    });
-
-  });
+  // describe('the user enters existing user credentials and submits the form', () => {
```
Comment (jans510) [inline src/app/features/home/components/login-form/login-form.component.spec.ts:59]: is this a leftover?

**Comment by jans510** [comment]
Looks good to me in general but I did not went through every detail :)

File: src/app/features/home/components/login-form/login-form.component.spec.ts (hunk 1 above)
Comment (halbekanne) [reply src/app/features/home/components/login-form/login-form.component.spec.ts:59]: yes

**Comment by halbekanne** [comment]
> Looks good to me in general but I did not went through every detail :)

That's ok, the important thing with this PR is that tests are passing and people can start creating them. The tests are fixed to work but otherwise mostly don't make a lot of sense cause almost no functionality is tested yet. So feel free to replace any tests with better ones whenever you happen to touch a component.

**Pull request merged**